### DISCUSSION TITLE
How to Modify type

### DISCUSSION THREAD

**Discussion by daiku60** [LekoArts/gatsby-themes#734] [Questions] [answered]
Hi! I'm trying to understand how to use the template and customize it. So far I've been able to shadow components, but the issue is that I need to embed images to a mdx file in pages using the frontmatter. 
As explained [here](https://www.gatsbyjs.com/blog/mdx-embedded-gatsby-image/) i need to add the field `embeddedImagesLocal: [File] @fileByRelativePath` to the Page/MdxPage type, and then take the value from the `node.frontmatter.embeddedImagesLocal` attribute, the same way the banner does. For that, i need to customize gatsby-config.js, but unfortunately I've been unable to do so, without clashing typeCreations. 
As discussed [here](https://stackoverflow.com/questions/62494409/how-do-you-shadow-gatsby-node-js-of-the-themes-theme), I cannot shadow files outside the src folder, so I cannot override the default behavior.

Is there any workaround on this? Thanks!

**Comment by LekoArts** [comment, marked as answer]
Hi!

Let me know if these two issues might help:
- https://github.com/LekoArts/gatsby-themes/issues/387
- https://github.com/LekoArts/gatsby-themes/issues/438

**Reply by daiku60** [reply]
Thanks! Just what I needed ^_^
